integration-tests/mongodb-client/src/test/java/io/quarkus/it/mongodb/BookResourceWithParameterInjectionTest.java

```diff
@@ -8,12 +8,9 @@
 import org.junit.jupiter.api.condition.DisabledOnOs;
 import org.junit.jupiter.api.condition.OS;
 
-import io.quarkus.test.common.QuarkusTestResource;
 import io.quarkus.test.junit.QuarkusTest;
-import io.quarkus.test.mongodb.MongoTestResource;
 
 @QuarkusTest
-@QuarkusTestResource(MongoTestResource.class)
```

Comment:
Why do you remove it ?

Reply:
Because `@QuarkusTestResource` applies to all test by default so:

* There is no reason to repeat the same resource in each test
* If we have two differently configured `@QuarkusTestResource` annotations, then two different instances will be started - in this case two different mongo instances

Reply:
I agree with your pesepective and that's why I've said multiple times that what we did with `@QuarkusTestResource` was a mistake - unfortunately there is no backing away from it now as it's too pervasive.